src/dmd/root/array.d

```diff
@@ -253,3 +253,63 @@ private:
     size_t len;
     size_t *ptr;
 }
+
+/**
+ * Converts the given root Array to the standard D array.
+ * Params:
+ *  array = the array to convert.
+ * Returns:
+ *  The given array converted to a standard D array.
+ */
+@property T[] asDArray(T)(ref Array!T array)
```

Comment:
I suggest rewording the comment to say "exposes" rather than "converts".

Reply:
sure